### ISSUE TITLE
[a11y audit] Finding: Chatbot MHV Prescription Skill: 4.3.7 Uses legible fonts

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#67209]
## 4.3.7 Uses legible fonts 
- **Testing Status:** `Completed`
- **Findings**: `Present`
- **Audit Spreadsheet:** [Link to 4.3.7](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=1502697987&range=A39)

## Testing Detail 

I tested for the following:

> The chatbot font(s) adheres to WCAG guidelines and does not include italics, block capitals, underlining, justified alignment, and serif fonts

### Does Not Pass 🔴 

The prescription skill includes **block capitals** in a number of places, so it doesn't pass.

### Recreating the Issue 🖥️ 

To view the issue, go to the prescription skill: https://dev.va.gov/contact-us/virtual-agent/
- Visit each of the three paths:
    - List prescriptions
    - Request refills
    - Track shipments.

### How to Fix 🛠️ 

These `BLOCK CAPITALS` need to be replaced with `Sentence case` (unless otherwise specified).

#### List prescriptions
The prescripton name in the chatbot message
![Screenshot 2023-10-10 at 1 35 15 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/58805c4b-3436-494f-8a47-30e555358b74)

The prescription name as the list view title
![Screenshot 2023-10-10 at 1 34 20 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/842cf4ee-5b72-4584-ba78-39cf7256f884)

#### Request refills
The prescription name, in the chatbot message and the list view
![Screenshot 2023-10-10 at 1 36 00 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/8f6c065d-dbd2-41ac-8a20-0aa3cda40903)

#### Track shipments
The prescription name, in the chatbot message and the list view
![Screenshot 2023-10-10 at 1 38 49 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/1e7103ab-a18e-4639-9d23-7f9a26fe83bc)

#### Facility name
The VA facility name (this should be written in plain language - the actual name of the facility, followed by the abbreviation in parentheses if that's helpful for the user) and "Other prescriptions in package"'s "YES" or "NO"
![Screenshot 2023-10-10 at 1 39 43 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/b943a94d-914a-4d81-8d03-9758842c9a1b)

#### Other items in package
The list of prescriptions in "Other items in package"
![Screenshot 2023-10-10 at 1 40 11 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/d9ec5ca6-f877-4858-84e9-da99983ee6c0)

**Comment by sara-amanda** [comment]
## Finalizing the Ticket 
@coforma-jamie This is a great way to document your testing for this project! Thank you for auditing this! 
- I went ahead and I added a summary section at the top linking to our spreadsheet.
- Added a testing details section with headings for recreating the issue, how to fix and issue locations. 
- **Then I renamed the ticket, now that it is done:** 
    - **From**: [a11y audit] `Testing`: Chatbot MHV Prescription Skill: 4.3.7 Uses legible fonts
    - **To**:  [a11y audit] `Finding`: Chatbot MHV Prescription Skill: 4.3.7 Uses legible fonts
- I also added the ticket number to your hyperlink in the spreadsheet. This is a great alternative to housing images in a folder, making the deliverable have one click to the ticket for more information.

**Comment by sara-amanda** [comment]
- [x] Closing Ticket
